spec/support/verified_double.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module VerifiedDouble
```

Comment:
Is this absolutely necessary? No other simpler way of doing it?

Reply:
For context, this was added because `case` statement fails for `instance_double` in `Command::Terraform::Import#redource_id:64`

Browsed through github discussion and decided to copy solution from here - https://github.com/rspec/rspec-mocks/issues/794#issuecomment-106607074 because it looks like more general solution